Backend/Models/Users/User.cs

```diff
@@ -67,6 +67,11 @@ public User(WebSocket socket, string ip)
             _ip = ip;
         }
 
+        public override string ToString()
+        {
+            return Username;
+        }
```

Comment:
@ahmed-okkio : Change the function name to something more indicative of what you're using this for something like ToArrayEncoderString()